src/diffusers/pipelines/kolors/tokenizer.py

```diff
@@ -143,10 +143,18 @@ def get_command(self, token):
     def unk_token(self) -> str:
         return "<unk>"
 
+    @unk_token.setter
+    def unk_token(self, value: str):
+        self._unk_token = value
```

Comment:
Seems like these should have been done in a separate PR?

Reply:
if I don't do this, the tests fails (throws AttributeError), I'm ok with doing this in a separate PR but then I'll have to skip the tests in this PR

Reply:
Ah okay. Then no worries.